railgun/manager/flagset.go

```diff
@@ -0,0 +1,17 @@
+package manager
+
+import (
+	"flag"
+
+	"github.com/kong/kubernetes-ingress-controller/pkg/util"
+)
+
+// flagSet extends flag.FlagSet with additional variable types.
+type flagSet struct {
+	flag.FlagSet
+}
+
+func (f *flagSet) EnablementStatusVar(p *util.EnablementStatus, name string, value util.EnablementStatus, usage string) {
```

Comment:
lint: public methods need godoc comment

(I'm starting to think we should modify our lint CI actually :thinking:)